spec/spec_helper.rb

```diff
@@ -1,14 +1,7 @@
-require 'rubygems'
 require 'rspec/autorun'
-require "active_record"
-require 'active_support'
+require 'rails/all'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.